src/Sulu/Bundle/ContentBundle/Tests/Functional/Repository/ResourceLocatorRepositoryTest.php

```diff
@@ -1,127 +0,0 @@
-<?php
-
-/*
- * This file is part of Sulu.
- *
- * (c) MASSIVE ART WebServices GmbH
- *
- * This source file is subject to the MIT license that is bundled
- * with this source code in the file LICENSE.
- */
-
-namespace Sulu\Bundle\ContentBundle\Tests\Functional\Repository;
-
-use Sulu\Bundle\ContentBundle\Document\PageDocument;
-use Sulu\Bundle\ContentBundle\Repository\ResourceLocatorRepositoryInterface;
-use Sulu\Bundle\TestBundle\Testing\SuluTestCase;
-use Sulu\Component\DocumentManager\DocumentManagerInterface;
-
-/**
- * @group functional
- * @group repository
- */
-class ResourceLocatorRepositoryTest extends SuluTestCase
-{
-    /**
-     * @var DocumentManagerInterface
-     */
-    private $documentManager;
-
-    /**
-     * @var ResourceLocatorRepositoryInterface
-     */
-    private $repository;
-
-    protected function setUp()
-    {
-        $this->initPhpcr();
-        $this->documentManager = $this->getContainer()->get('sulu_document_manager.document_manager');
-        $this->repository = $this->getContainer()->get('sulu_content.rl_repository');
-    }
-
-    public function testGenerate()
-    {
-        /** @var PageDocument $document */
-        $document = $this->documentManager->create('page');
-        $document->setTitle('test');
-        $document->setResourceSegment('/test');
-        $document->setStructureType('overview');
-        $this->documentManager->persist($document, 'en', ['parent_path' => '/cmf/sulu_io/contents']);
-        $this->documentManager->publish($document, 'en');
-        $this->documentManager->flush();
-
-        $result = $this->repository->generate(
-            [
-                'title' => 'test',
-            ],
-            $document->getUuid(),
-            null,
-            'sulu_io',
-            'en',
-            'overview'
-        );
-
-        $this->assertEquals('/test/test', $result['resourceLocator']);
-    }
-
-    public function testGenerateSlash()
```

Comment:
@danrot: This should be in the same equivalence partation as testGenerate(). 
The slash replacement is implemented in the RlpStrategy which is mocked.